src/module/rules/rule-element/special-resource.ts

```diff
@@ -104,6 +117,34 @@ class SpecialResourceRuleElement extends RuleElementPF2e<SpecialResourceSchema>
         }
     }
 
+    /** Treat special resources as upgrades during the AELike phase */
+    override onApplyActiveEffects(): void {
+        if (this.ignored) return;
```

Comment:
I guess this should be responsive to a predicate, though I can't envision a use case at the moment.